opentelemetry-sdk/tests/trace/test_sampling.py

```diff
@@ -66,7 +70,7 @@ def test_always_off(self):
             0xDEADBEF2,
             "sampled parent, sampling off",
         )
-        self.assertFalse(sampled_always_on.sampled)
+        self.assertFalse(sampling.is_sampled(no_record_always_off.decision))
```

Comment:
looks like this is changing the test, should this be `sampled_always_ok.decision`?